ui/hooks/accounts/useMultichainClient.ts

```diff
@@ -35,18 +57,14 @@ export class BitcoinWalletSnapClient {
       scope,
     });
 
-    // NOTE: The account's balance is going to be tracked automatically on when the new account
-    // will be added to the Snap bridge keyring (see `BalancesController:#handleOnAccountAdded`).
-    // However, the balance won't be fetched right away. To workaround this, we trigger the
-    // fetch explicitly here (since we are already in a `async` call) and wait for it to be updated!
     await multichainUpdateBalance(account.id);
   }
 }
 
-export function useBitcoinWalletSnapClient() {
+export function useMultichainClient(clientType: WalletClientType) {
```

Comment:
Related to my previous comment, this would be: `useMultichainWalletSnapClient` here too